// menu-lib/src/styles/Menu.js
import { Link } from 'react-router-dom';
import styled from 'styled-components';

export const SidebarLink = styled(Link)`
display: flex;
color: #edeff3;
justify-content: space-between;
align-items: center;
padding: 20px;
list-style: none;
height: 30px;
text-decoration: none;
font-size: 14px;
line-height: 3.2em;

  &:hover {
    text-decoration: underline;
    border-left: double #edeff3;
  }
`;

export const SidebarNav = styled.nav`
  background-color: #2f353a;
  width: 250px;
  height: 100vh;
  display: flex;
  justify-content: center;
  position: relative;
  top:0;
  left: 0;
  transition: 350ms;
  z-index: 10;
`;

export const SidebarWrap = styled.div`
  width: 100%;
  height: 2.5em;
`;

export const DropdownLink = styled(Link)`
  background: #414757;
  height: 50px;
  padding-left: 2rem;
  display: flex;
  align-items: center;
  text-decoration: none;
  color: #f5f5f5;
  font-size: 14px;

    &:hover {
      text-decoration: underline;
    }
`;

export const SidebarLaber = styled.label`
  margin-left: 10px;
`;
